src/components/NotificationModal.tsx

```diff
@@ -93,37 +92,25 @@ export const NotificationModal: React.FC<ModalProps> = ({
   const targetPath = get(currentMethodControl, "targetPath");
 
   return (
-    <Modal
+    <StyledModal
+      centered
       data-test-id="subscription-modal"
-      title={<H2>Add Subscription</H2>}
+      footer={null}
       visible={visible}
       onCancel={onCancel}
-      footer={[
-        <Button
-          key="cancel"
-          onClick={onCancel}
-          dataCy="cancel-subscription-button"
-        >
-          Cancel
-        </Button>,
-        <Button
-          key="save"
-          dataCy="save-subscription-button"
-          loading={mutationLoading}
-          disabled={!isFormValid}
-          onClick={onClickSave}
-          variant="danger"
-        >
-          Save
-        </Button>,
-      ]}
       width="50%"
       wrapProps={{
         "data-cy": "task-notification-modal",
       }}
     >
-      <>
-        when
+      <ModalTitle>Add Subscription</ModalTitle>
```

Comment:
I think you can just remove the `ModalTitle` styling and just pass the H2 directly to the title prop in the modal since thats what we do for some other modals

Reply:
the ModalTitle component takes care of spacing before an after the line so I think it's necessary. here is what it looks like using H2 directly. 
<img width="460" alt="Screen Shot 2020-06-18 at 3 21 55 PM" src="https://user-images.githubusercontent.com/10734386/85063034-85aaa880-b177-11ea-9bb5-6c40c77cadec.png">

Reply:
I was referring to doing something like this. 
![image](https://user-images.githubusercontent.com/4605522/85324384-be0ff680-b497-11ea-8536-b32242946739.png)
![image](https://user-images.githubusercontent.com/4605522/85324410-cbc57c00-b497-11ea-9fcb-42ef201c9e97.png)

Reply:
when using the title prop, we don't have control over the padding and color for the line underneath the title so to match the modal designs I had to pass the title component into the body of the modal component
from the design:
![Screen Shot 2020-06-23 at 11 11 43 AM](https://user-images.githubusercontent.com/10734386/85421381-5bc00000-b542-11ea-857d-292acbbbe325.png)
.